packages/@storefront/infinite-scroll/src/infinite-scroll/index.ts

```diff
@@ -353,7 +353,7 @@ class InfiniteScroll {
   };
 
   replaceState = () => {
-    if (!this.state.firstLoad) this.flux.replaceState(this.state.route);
+    if (!this.state.firstLoad) this.flux.replaceState(this.state.route, true);
```

Comment:
I'm surprised the linter didn't complain about this.

Reply:
Sorry I'm not sure what you're referring to. Needs `{}`?

Reply:
I was expecting it to say that you should reformat it like so:
```js
if (!this.state.firstLoad) {
  this.flux.replaceState(this.state.route, true);
}
```

I'm personally okay with this, though, so I won't ask you to change it.